### PR TITLE
Add node script to promote lambda to staging

### PR DESCRIPTION
**Pull request by TCbacon**
A node script to promote lambda to staging locally using `shelljs` module. 
The commands are similar to what is in the github actions `build_api.yml` and `deploy_api.yml files`.
The `lambdatostaging.js` file is the main driver and `lambdatostaginghelper.js` file contain the build and deploy logic.
Navigate to `APILambda` directory and type in command: `npm run deploylambda`.

### CODE REVIEW

File: APILambda/lambdatostaging/lambdatostaginghelper.js [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    deleteDistFolderContent(dir = TIFBACKENDUTILS_DIR) {
+        let result;
+
+        if (!fs.existsSync(dir)) {
+            console.log(`${dir} does not exist.`);
+            return;
+        }
+
+        console.log(`Deleting all contents from ${dir}/dist`);
+        result = shell.rm('-rf', `${dir}/dist/*`);
```
Comment (TCbacon) [inline APILambda/lambdatostaging/lambdatostaginghelper.js]: parameterize shelljs command

**Comment by seanim0920** [comment]
[Make a node script to promote local lambda to staging](https://trello.com/c/aRSUHJHp/534-make-a-node-script-to-promote-local-lambda-to-staging)

**Comment by seanim0920** [comment]
[Make a node script to promote local lambda to staging](https://trello.com/c/aRSUHJHp/534-make-a-node-script-to-promote-local-lambda-to-staging)

**Comment by seanim0920** [comment]
We'll use this, this should be fine https://github.com/tifapp/FitnessProjectBackend/pull/216

**Pull request closed without merging**